neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/http/impl/HttpNeo4jClient.java

```diff
@@ -25,7 +30,11 @@
  *
  * @author Davide D'Alto
  */
-public class RemoteNeo4jClient implements AutoCloseable {
+public class HttpNeo4jClient {
```

Comment:
No particular reason, I guess I didn't notice I removed it because we are not using it anywhere in our code.